src/Stripe/Services/Customers/StripeCustomerCreateOptions.cs

```diff
@@ -28,6 +28,9 @@ public class StripeCustomerCreateOptions
         [JsonProperty("plan")]
         public string PlanId { get; set; }
 
+        [JsonProperty("tax_percent")]
+        public decimal? TaxPercent { get; set; }
```

Comment:
What do you think about this pull request? It's all officially supported by Stripe. I have also tested all this, and it works perfectly. Could you please get it merged for the next release? Thanks